relevanceai/vector_tools/cluster.py

```diff
@@ -5,14 +5,16 @@
 import warnings
 
 from typing import List, Union, Dict, Any, Optional
-
 from doc_utils import DocUtils
+from joblib.memory import Memory
 
 from relevanceai.api.client import BatchAPIClient
 from relevanceai.logger import LoguruLogger
 from relevanceai.vector_tools.constants import CLUSTER, CLUSTER_DEFAULT_ARGS
 from relevanceai.errors import ClusteringResultsAlreadyExistsError
 
+EXISTING_CLUSTER_MESSAGE = """Clustering results already exist"""
```

Comment:
Instead of setting it as a global parameter, can we just make this the default message in the Exception instead?